Image Downloader/ImageDownload/ImageDownloadViewController.swift

```diff
@@ -35,9 +35,6 @@ class ImageDownloadViewController: UIViewController {
             action: #selector(downloadButtonTapped),
             for: .touchUpInside
         )
-        setConstraints()
-        print/bye-world
-        print("Bye bye World.")
```

Comment:
Keep this line of code if only updated the background color to orange.

Reply:
This code crashes the app too and the code below that does nothing, why not delete

Reply:
setConstraints() and print/bye-world can be deleted.
Please makes comment if you deleting the print statement so people know that you delete this line of code on purpose.